cmd/updater/update.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
To be honest, I'm not sure whether this is the correct bash conventions, but isn't it generally in a separate line on a bash file as `set -e`?

Reference: https://stackoverflow.com/questions/43116895/standard-shebang-with-e

Reply:
Also CircleCI tests seem to be failing on this PR

Reply:
+1 the convention in our scripts uses `set` and `/usr/bin/env bash`.